index/05-conclusion.Rmd

```diff
@@ -0,0 +1,9 @@
+# Conclusion {-}
+
+At the end of this internship, the progress of the project is great. Principal data have been recovered to be integrated as explanatory variables and first models have been built. Now, the routine is ready to build further models. First results are encouraging because models are relatively relevant and they can be improved adding new explanatory variables and using other statistical methods. Indeed, comparing models with a lot of different methods will be possible to find the best method, i.e. the method with the smallest error (RMSE).
```

Comment:
- Now, the routine is ready to build further models that could make use of other learning algorithms as various forms of kriging, ANN, etc.
-  Indeed, comparing models with a lot of different methods will be possible to find the best method, i.e. the method with the smallest error (RMSE). ==> I would remove this